src/video/camerasource.cpp

```diff
@@ -214,14 +233,18 @@ void CameraSource::unsubscribe()
     }
     else
     {
+        //qDebug() << "second block";
         device->close();
         biglock = false;
     }
+    subscriptions--;
```

Comment:
Ok.
I didn't know about clang SCA in qtcreator. Thx for news. Maybe it's using have to be included into project coding rules.